tests/test_helper.go

```diff
@@ -0,0 +1,212 @@
+package tests
+
+import (
+	"context"
+	migapi "github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/kubernetes"
+	"os"
+	"strings"
+)
+
+// common constants
+const (
+	TestObjectName         = "foo"
+	TestNamespace          = "openshift-migration"
+	TestMigrationNamespace = "ocp-41583-longpvcname"
```

Comment:
do these need to be constants? I would imaging for each name we would have to find a way to get the names of such objects derived from the test name?

Reply:
Agreed, removed the constants, for now we can pass it to function and get the `migmigartion` and `migplan` with passed name